system/modules/isotope/library/Isotope/Module/ProductList.php

```diff
@@ -151,7 +151,7 @@ protected function compile()
         if (!\is_array($arrProducts)) {
             // Display "loading products" message and add cache flag
             if ($this->blnCacheProducts) {
-                $blnCacheMessage = (bool) $this->iso_productcache[$cacheKey];
+                $blnCacheMessage = $this->iso_productcache[$cacheKey] ?? false;
```

Comment:
I think we should keep the boolean casting

Reply:
Sure, why not, but isn't `iso_productcache` always an array of boolean values anyway? Also, `$blnCacheMessage` is only used in `if` statement, which casts it to bool automatically.